third_party/ScubaDiver/src/Diver.cs

```diff
@@ -90,32 +84,14 @@ public Diver()
       {"/get_item", MakeArrayItemResponse},
     };
     _remoteEventHandler = new ConcurrentDictionary<int, RegisteredEventHandlerInfo>();
-    _unifiedAppDomain = new UnifiedAppDomain(this);
-  }
-
-  #region Bootstrapper Cleanup
-
-  private static bool UnloadBootstrapper()
-  {
-    foreach(ProcessModule module in Process.GetCurrentProcess().Modules)
-    {
-      if (new string[] {
-          "Bootstrapper.dll",
-          "Bootstrapper_x64.dll"
-        }.Any(s => module.ModuleName == s))
-      {
-        return Kernel32.FreeLibrary(module.BaseAddress);
-      }
-    }
-    return false;
   }
 
-  #endregion
-
   public void Start(ushort listenPort)
   {
+    // Start the ClrMD runtime
+    _runtime = new SnapshotRuntime();
+
     // Start session
-    RefreshRuntime();
     HttpListener listener = new();
     string listeningUrl = $"http://127.0.0.1:{listenPort}/";
     listener.Prefixes.Add(listeningUrl);
```

Comment:
Somehow we've hit a regression where client registration is currently broken; ClrMD injection works fine and ScubaDiver redirects logging to a console in debug mode (from the dll entrypoint).

It seems that the diver thread isn't blocking the dll entrypoint:
![image](https://github.com/videre-project/MTGOSDK/assets/32466081/5c7a9331-8858-4ccb-ad10-c02832714213)